sos/report/plugins/aap_containerized.py

```diff
@@ -107,6 +107,19 @@ def setup(self):
                     subdir="aap_containers_log"
             )
 
+        if "automation-controller-task" in aap_containers:
+            container = "automation-controller-task"
+            podman_commands = [
+                (f"su - {username} -c 'podman exec -it {container} bash -c"
+                 " \"awx-manage check_license --data\"'",
+                 "awx-manage_check_license_--data"),
+                (f"su - {username} -c 'podman exec -it {container} bash -c"
+                 " \"awx-manage list_instances\"'",
+                 "awx-manage_list_instances"),
+            ]
+            for command, filename in podman_commands:
+                self.add_cmd_output(command, suggest_filename=filename)
```

Comment:
Does this not work using the `runas` and `container` parameters for `add_cmd_output()`?

E.G.

```python
podman_commands = [
    'awx-manage check-license --data'
    ...etc...
]
for command in podman_commands:
    self.add_cmd_output(command, suggest_filename=command, runas=username, container=container)
```

Reply:
Hmm, guess we should look into that. I have an idea as to why that's happening but it'd take a bit to address it and I don't want that to hold this up any further.

ack, with a note on me to look into the `runas` and `container` combination.